app/template/assets/cui-2.0.0/js/vkaci.js

```diff
@@ -273,11 +273,10 @@ function draw_leaf() {
 
 function draw_node() {
     var str = $("#nodename").val();
-    var asnPresent = $("#asnPresent").val();
     if (!str.trim()) return;
     var seed = "0.7578607868826415:1645663636870"
     // var config_node = neo_viz_config(true, "viz_node", 'MATCH (p:Pod)-[r]->(n:Node)-[r1*1..3]->(m) WHERE n.name= "' + str + '" RETURN *', seed)
-    if (asnPresent) {
+    if (asnPresent === 'True') {

```

Comment:
You can template in the value of asnPresent as a Boolean instead of a string. This would increase the readability of the JS code and we wouldn’t be doing a string comparison.